lib/iris/analysis/maths.py

```diff
@@ -332,7 +332,10 @@ def divide(cube, other, dim=None, in_place=False):
     _assert_is_cube(cube)
     other_unit = getattr(other, 'units', '1')
     new_unit = cube.units / other_unit
-    op = operator.idiv if in_place else operator.div
+    try:
+        op = operator.itruediv if in_place else operator.truediv
+    except AttributeError:
+        op = operator.idiv if in_place else operator.div
     return _binary_op_common(op, 'divison', cube, other, new_unit, dim,
```

Comment:
> I don't know what that second parameter is used for though.

It's used to customise various errors messages in `_assert_matching_units` and `_broadcast_cube_coord_data`. From the error that begins "Could not determine dimension..." it looks like the best value for this string would be the name of the function, i.e. "divide".